jcl/src/java.base/share/classes/java/lang/Throwable.java

```diff
@@ -362,7 +361,16 @@ public String toString () {
  * @return		the receiver.
  */
 public synchronized Throwable initCause(Throwable throwable) {
-	return	setCause(throwable);
+	/*[PR CMVC 199629] Exception During Class Initialization Not Handled Correctly */
```

Comment:
Might as well remove this line since its not relevant to OpenJ9, and no longer seems relevant to anyone

Reply:
Are you suggesting I remove all `/*[PR ... */` comments or just this one?

Reply:
Just this one since the code was modified anyway.

Reply:
If you don't want to remove it that is fine. I don't have any other comments. Ready to run PR testing once the WIP is removed.

Reply:
I don't mind removing it, I just wasn't clear on the scope of your request.
I'll squash that and the cmake change into related commits and then remove the WIP.